specification/maps/data-plane/Microsoft.Maps/Weather/preview/1.0/weather.json

```diff
@@ -702,6 +709,203 @@
+          {
+            "$ref": "#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionKey"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/JsonFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherLanguage"
+          },
+          {
+            "name": "query",
+            "description": "Coordinates of the location for which air quality is requested. The applicable query is specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
+            "in": "query",
+            "type": "string",
+            "required": true
+          },
+          {
+            "$ref": "#/parameters/Pollutants"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CurrentAirQualityResponse"
+            }
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "401": {
+            "$ref": "#/responses/401"
+          },
+          "403": {
+            "$ref": "#/responses/403"
+          },
+          "404": {
+            "$ref": "#/responses/404"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        }
+      }
+    },
+    "/weather/airQuality/forecast/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Air Quality Forecast**\n\n**Applies to**: S1 pricing tiers.\n\nThe service provides the detailed information about the concentration of pollutant and overall status for upcoming few days air quality. The service can provide forecast air quality for future one to four days.",
+        "operationId": "Weather_GetDailyAirQualityForecast",
+        "x-ms-examples": {
+          "DailyAirQualityForecast": {
+            "$ref": "./examples/GetDailyAirQuality.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionKey"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/JsonFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherLanguage"
+          },
+          {
+            "name": "query",
+            "description": "Coordinates of the location for which air quality is requested. The applicable query is specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
+            "in": "query",
+            "type": "string",
+            "required": true
+          },
+          {
+            "name": "duration",
+            "description": "Specifies for how many days from now we would like to know about the air quality. Available values are 1 ~ 4.",
```

Comment:
Per below AccuWeather API doc, the available time spans are 1, 2, 3, 4, 5, 6, and 7 days.
http://apidev.accuweather.com/developers/airQualityAPIParameters#queryParameters

Reply:
Thanks for the remind!
It's kind of surprising that our current implementation only allows 1 to 4 which we also need to take care of in the near future.
https://msazure.visualstudio.com/One/_git/Azure-IoT-LocationBasedServices?path=%2Fsrc%2FALSProxy%2FALS%2FModels%2FWeather%2FDailyForecastAirQualityRequest.cs&_a=contents&version=GBmaster